packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -408,25 +409,27 @@ export class RunningSummarizer implements IDisposable {
     }
 
     /** {@inheritdoc (ISummarizer:interface).summarizeOnDemand} */
-    public summarizeOnDemand({
-        reason,
-        ...options
-    }: IOnDemandSummarizeOptions): OnDemandSummarizeResult {
+    public summarizeOnDemand(
+        resultsBuilder: OnDemandSummarizeResultBuilder | undefined = new OnDemandSummarizeResultBuilder(),
```

Comment:
I think undefined does not do anything here, worth dropping (maybe I'm wrong RE how TSC deals with optional args with default values)